stripe/resource.py

```diff
@@ -312,9 +312,32 @@ def instance_url(self):
 
 class ListObject(StripeObject):
 
-    def all(self, **params):
+    def list(self, **params):
         return self.request('get', self['url'], params)
 
+    def all(self, **params):
+        warnings.warn("The `all` method is deprecated and will"
+                      "be removed in future versions. Please use the "
+                      "`list` method instead",
+                      DeprecationWarning)
+        return self.list(**params)
+
+    def paging_iter(self):
```

Comment:
Bikeshed: I think somebody at one point pitched the idea of this being named `auto_paging_iter` (@russelldavis?). That might be worth considering just for the sake of consistency with the Ruby bindings.